.travis.yml

```diff
@@ -24,3 +24,4 @@ script:
 #  - catkin run_tests
 #  - source devel/setup.bash
 #  - devel/lib/behaviours/behaviour_test
+   - testing/run_tests.sh
```

Comment:
This breaks the travis.yml file. I think this line cannot be separated from above by comments (clobbers the whitespace).